lib/broadside/cli/global.rb

```diff
@@ -34,6 +41,14 @@ def call_hook(type, command)
 
 pre do |global, command, options, args|
   Broadside.load_config(global[:config])
+
+  if global[:debug]
```

Comment:
this is saying if `--debug` is set, ignore loglevel settings and force `Logger::DEBUG`